<launch>
    <!-- Opti Track -->
    <node pkg="optitrack_vrpn" type="node"
        name="optitrack_vrpn_node" output="screen">
    </node>

    <!-- CVO Server -->
    <node pkg="crazyflie" type="cvo_server"
        name="cvo_server" output="screen">
    </node>


    <!-- Mocap Estimation -->
    <node name="estimator1" type="ekf_ros" pkg="dragonfly" output="screen">
      <param name="id" value="1"/>
    </node>

    <!-- Mocap Estimation -->
    <!-- <node name="estimator2" type="ekf_ros" pkg="dragonfly" output="screen">
      <param name="id" value="2"/>
    </node> -->

</launch>
